server/src/main/java/org/opensearch/search/aggregations/bucket/histogram/InternalDateHistogram.java

```diff
@@ -452,11 +499,16 @@ private void addEmptyBuckets(List<Bucket> list, ReduceContext reduceContext) {
                 key = nextKey(key).longValue();
             }
         }
+        int postAddEmptyBucketCount = list.size() - estimateEmptyBucketCount - originalSize;
+        if (postAddEmptyBucketCount > 0) {
+            reduceContext.consumeBucketsAndMaybeBreak(postAddEmptyBucketCount);
+        }
```

Comment:
I remember we can account for negative value here. 
https://github.com/opensearch-project/OpenSearch/blob/f7b31b05f0bfe2aec1ceac387329dc71dc14021a/server/src/main/java/org/opensearch/search/aggregations/MultiBucketConsumerService.java#L165-L203

Reply:
This part I don't understand? why would we want to add negative bucket values?